### PR TITLE
This version of DocusignPlugin uses maven build and JWT Auth 2.0

### PR DESCRIPTION
**Pull request by emshedu**
Due to the dependencies needed, we are building the docusignPlugin.jar with maven. The remaining file structures are the same

### CODE REVIEW

**Review by ggarcia-ibm — APPROVED** [review]

File: DocuSignSamplePlugin/DocuSignPlugin/pom.xml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  <version>2.0</version>
+  <name>DocuSignPlugin</name>
+
+  <dependencies>
+    <!-- Navigator-->
+    <dependency>
+      <groupId>com.ibm.ecm.icn</groupId>
+      <artifactId>navigatorAPI</artifactId>
+      <version>[3.0.11-0,)</version>
+      <scope>provided</scope>
```
Comment (andychoi-ibm) [inline DocuSignSamplePlugin/DocuSignPlugin/pom.xml:16]: This should have <systemPath> as navigatorAPI won't be available via Maven repository.

File: DocuSignSamplePlugin/DocuSignPlugin/pom.xml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      <groupId>javax.mail</groupId>
+      <artifactId>mail</artifactId>
+      <version>1.4.7</version>
+      <scope>provided</scope>
+    </dependency>
+    <!-- taskManagerAPI-->
+    <dependency>
+      <groupId>com.ibm.ecm.taskman</groupId>
+      <artifactId>taskManagerAPI</artifactId>
+      <version>3.0.6-SNAPSHOT</version>
```
Comment (andychoi-ibm) [inline DocuSignSamplePlugin/DocuSignPlugin/pom.xml:55]: This should have as taskManagerAPI won't be available via Maven repository.

File: DocuSignSamplePlugin/DocuSignPlugin/src/main/java/com/ibm/icn/extension/docusign/DocuSignPlugin.java [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+    @Override
+    public String getCSSFileName() {
+        return "DocuSignPlugin.css";
+    }
+
+    @Override
+    public String getCopyright() {
+        /* Note: The copyright is not translated, so do not put into properties
+         * files that will be translated. */
+        String copyright = "Licensed Materials - Property of IBM Corp. (5725-U01) "
```
Comment (andychoi-ibm) [inline DocuSignSamplePlugin/DocuSignPlugin/src/main/java/com/ibm/icn/extension/docusign/DocuSignPlugin.java:43]: Are we licensing this?

File: DocuSignSamplePlugin/DocuSignPlugin/src/main/java/com/ibm/icn/extension/docusign/service/DocuSignConfiguration.java [hunk 4]
```diff
@@ -14,15 +14,14 @@
 /*****************************************************************************************************************
  * This class stores the DocuSign credentials required to connect to DocuSign systems.
  * The credentials required to authenticate to a DocuSign system are 
- * 				userName
- * 				password
+ * //Deprecated.
```
Comment (andychoi-ibm) [inline DocuSignSamplePlugin/DocuSignPlugin/src/main/java/com/ibm/icn/extension/docusign/service/DocuSignConfiguration.java:17]: Are we deprecating this class? You can use @Deprecated to indicate that.

File: DocuSignSamplePlugin/DocuSignPlugin/src/main/java/com/ibm/icn/extension/docusign/service/DocuSignConfiguration.java [hunk 5]
```diff
@@ -14,15 +14,14 @@
...
+ * //TODO: Migrate TO OAUTH2.0
```
Comment (andychoi-ibm) [inline DocuSignSamplePlugin/DocuSignPlugin/src/main/java/com/ibm/icn/extension/docusign/service/DocuSignConfiguration.java:18]: Is this done or still a todo item?

Comment (emshedu) [reply DocuSignSamplePlugin/DocuSignPlugin/src/main/java/com/ibm/icn/extension/docusign/service/DocuSignConfiguration.java:18]: Hi Andy, Thanks for the catch, Making those changes now

File: DocuSignSamplePlugin/DocuSignPlugin/src/main/java/com/ibm/icn/extension/docusign/DocuSignPlugin.java (hunk 3 above)
Comment (emshedu) [reply DocuSignSamplePlugin/DocuSignPlugin/src/main/java/com/ibm/icn/extension/docusign/DocuSignPlugin.java:43]: no

**Pull request merged**